content/ios/getting-started.md

```diff
@@ -12,66 +12,52 @@ This guide will show you how to integrate Skygear into an existing iOS project.
 <a name="sign-up-hosting"></a>
 ## Signing up for Skygear Hosting
 
-Sign up an account at [Skygear Portal](http://portal.skygear.io/).
+To start using the Skygear JS SDK, you need to register your account and
+application at the Skygear [Developer Portal](https://portal.skygear.io)
+website. After you registered, go to the **INFO** tab and copy down your
+`Server EndPoint` and `API Key`.
 
-You will need the _Server Endpoint_ and _API Key_ to set up your app.
-
-Go to the Info tab on the sidebar, and you will see your _Server Endpoint_ in the "Server Details" section and your _API Key_ in its section.
+[![Screenshot: where to look for Server EndPoint and API Key](/assets/common/portal-endpoint-apikey.png)](/assets/common/portal-endpoint-apikey.png)
 
 
 <a name="include-ios-sdk"></a>
-## Include the SDK in your project
-
-The installation requies [CocoaPods](https://cocoapods.org/).
-
-To install the Skygear iOS SDK as your iOS application dependency:
+## Including SDK in existing project
 
-### For Objective-C Projects
+The installation requies Xcode and [CocoaPods](https://cocoapods.org/). If you haven't installed them already, please head to [this section](#new-project) to read about how to craete a new project with configured SDK.
 
-1. Open and edit the `Podfile` file, add the following in the last line:
+### Step 1: Install SDK using CocoaPods
 
-```
-pod "SKYKit"
-```
+To install the Skygear iOS SDK as your iOS application dependency:
 
-2. Run `pod install` in your terminal.
-3. It's done! You now have installed Skygear SDK in your app.
+1. You need  to close Xcode.
+2. Open Terminal and navigate to the directory that contains your iOS project by using the cd command: `cd ~/Path/To/Your/App`. You can just drag the file icon to Terminal and the path will be automatically typed for you.
```

Comment:
- Please wrap the lines at 80 characters.
- Do we need these kinds of instructions? What are your thoughts? @david90 @chpapa @rickmak @cheungpat @ben181231